notebooks/en/agent_data_analyst.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>This is very cool, I'd end with saying that this is just a naive attempt and paint a future picture of what all is possible with the these going forward.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>